Objects/codeobject.c

```diff
@@ -1520,6 +1520,15 @@ PyCode_GetFreevars(PyCodeObject *code)
     return _PyCode_GetFreevars(code);
 }
 
+int
+_PyCode_GetNumFrameSlots(PyCodeObject *code)
```

Comment:
This should probably be a static inline function.

Reply:
Yeah. I also think it belongs in pycore_frame.h rather than pycore_code.h.